README.md

```diff
@@ -9,11 +9,16 @@ Sparkplug is a lightweight AMQP message consumer. It allows you to specify queue
 run tests with `nosetests`
 
 ## Releasing
+We release two versions: One to the FreshBooks internal PyPI repository, and one to the public repository.
```

Comment:
If the pypi release process works and is documented, why maintain both? We use many other python packages internally that don't get pushed directly to our private index, why should it be different for open source projects we maintain?

Reply:
Maintaining both I believe his more of a historical artifact. I don't see why we can't rip out pushing to the internal PyPI repository. Will you up for doing so @DeanWay? I wanted to document what existed as of today